python/src/scipp/table.py

```diff
@@ -7,6 +7,9 @@
 from .utils import value_to_string, name_with_unit
 from ._scipp import core as sc
 
+import ipywidgets as widgets
```

Comment:
Is there a risk that someone using a Jupyter notebook does not have or want ipywidgets? The new solution would break this for them then.

As in another open bug, this also adds a dependency of scipp on ipywidgets, even this is not used.